api/s2n.h

```diff
@@ -1964,7 +1964,7 @@ S2N_API extern int s2n_connection_free(struct s2n_connection *conn);
  *
  * Once `s2n_shutdown` is complete:
  * * The s2n_connection handle cannot be used for reading for writing.
- * * The underlying transport can be closed. Most likely via `close()`.
+ * * The underlying transport can be closed. Most likely via `shutdown()` or `close()`.
```

Comment:
We use "underlying transport" because the RFC uses that phrase. Everyone (or at least almost everyone) is probably using TCP in practice, but technically the RFC only requires "a reliable, in-order data stream". I'm not sure what phrase would be more understandable to people who haven't read the RFC while still being accurate :/

Reply:
Maybe "the underlying stream" or "the underlying channel" if it's "transport" that's confusing?

Reply:
I was thinking more like, if we're telling people to use shutdown() and close() then maybe it's more clear to say "the underlying socket can be closed." But I see the value in mirroring the RFC.